internal/uvm/constants.go

```diff
@@ -29,9 +31,43 @@ const (
 	WCOWGlobalMountPrefix = "C:\\mounts\\m%d"
 	// RootfsPath is part of the container's rootfs path
 	RootfsPath = "rootfs"
+
+	// Maximum number of SCSI controllers allowed
+	MaxSCSIControllers = 4
 )
 
 var (
 	errNotSupported = errors.New("not supported")
 	errBadUVMOpts   = errors.New("UVM options incorrect")
+
+	// V5 GUIDs for SCSI controllers
+	// These GUIDs are created with namespace GUID "d422512d-2bf2-4752-809d-7b82b5fcb1b4"
+	// and index as names. For example, first GUID is created like this:
+	// guid.NewV5("d422512d-2bf2-4752-809d-7b82b5fcb1b4", []byte("0"))
+	SCSI_CONTROLLER_GUIDS = []guid.GUID{
+		{
```

Comment:
Suggest we add a comment above each one of these with the GUID in written form (`d422512d-2bf2-4752-809d-7b82b5fcb1b4`), so searching for that GUID in the code is easier.